src/sentry/models/organization.py

```diff
@@ -232,7 +232,7 @@ def merge_to(from_org, to_org):
                     slug=project.slug,
                 )
 
-        for model in (ApiKey, AuditLogEntry):
+        for model in (ApiKey, AuditLogEntry, Release):
```

Comment:
we don't yet have a unique constraint (because of legacy data) but good question. i guess probably what we should do is just pick one of the releases? i guess this gets back to our earlier question of how are we going to migrate existing duplicates, so maybe i should add a todo here and make sure to handle that when we decide how we're merging existing duplicates